<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>600</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>RetroView</string>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::ForceTabbedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="CGLViewer" name="glView" native="true">
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>896</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionLoad_Basepath"/>
    <addaction name="actionLoadPak"/>
    <addaction name="actionExport"/>
   </widget>
   <widget class="QMenu" name="menu_Animations">
    <property name="title">
     <string>&amp;UV Animations</string>
    </property>
    <addaction name="actionMode0"/>
    <addaction name="actionMode1"/>
    <addaction name="actionMode2"/>
    <addaction name="actionMode3"/>
    <addaction name="actionMode4And5"/>
    <addaction name="actionMode6"/>
    <addaction name="actionMode7"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Animations"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad_Basepath"/>
   <addaction name="actionLoadPak"/>
   <addaction name="actionExport"/>
   <addaction name="actionEnableTextures"/>
   <addaction name="actionDrawPoints"/>
   <addaction name="actionDrawJointNames"/>
   <addaction name="actionDrawBoundingBox"/>
   <addaction name="actionDrawCollision"/>
   <addaction name="actionWireframe"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="infoDockWidget">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Info</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="4" column="0" colspan="2">
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabPosition">
        <enum>QTabWidget::West</enum>
       </property>
       <property name="documentMode">
        <bool>false</bool>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="fpsLabel">
       <property name="text">
        <string>FPS</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="resetCameraButton">
       <property name="text">
        <string>Reset Camera</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="materialSetComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="materialSetLbl">
       <property name="text">
        <string>Material Set</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="horizontalSlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Speed</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionExport">
   <property name="text">
    <string>&amp;Export</string>
   </property>
  </action>
  <action name="actionMode0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mode &amp;0</string>
   </property>
  </action>
  <action name="actionMode1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mode &amp;1</string>
   </property>
  </action>
  <action name="actionMode2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mode &amp;2</string>
   </property>
  </action>
  <action name="actionMode3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mode &amp;3</string>
   </property>
  </action>
  <action name="actionMode4And5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mode &amp;4/&amp;5</string>
   </property>
  </action>
  <action name="actionMode6">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mode &amp;6</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionLoadPak">
   <property name="text">
    <string>Load &amp;Pak</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionMode7">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mode &amp;7</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionEnableTextures">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable &amp;Textures</string>
   </property>
   <property name="toolTip">
    <string>Enable Textures</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionDrawPoints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Draw &amp;Points</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionDrawJointNames">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Draw Joint Names</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionDrawBoundingBox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Draw Bounding Box</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionWireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wireframe</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionDrawCollision">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Draw Collision</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionLoad_Basepath">
   <property name="text">
    <string>Load &amp;Basepath</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CGLViewer</class>
   <extends>QWidget</extends>
   <header>ui/CGLViewer.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExport</sender>
   <signal>triggered()</signal>
   <receiver>CMainWindow</receiver>
   <slot>onExport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMode0</sender>
   <signal>toggled(bool)</signal>
   <receiver>CMainWindow</receiver>
   <slot>onToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMode1</sender>
   <signal>toggled(bool)</signal>
   <receiver>CMainWindow</receiver>
   <slot>onToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMode2</sender>
   <signal>toggled(bool)</signal>
   <receiver>CMainWindow</receiver>
   <slot>onToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMode3</sender>
   <signal>toggled(bool)</signal>
   <receiver>CMainWindow</receiver>
   <slot>onToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMode4And5</sender>
   <signal>toggled(bool)</signal>
   <receiver>CMainWindow</receiver>
   <slot>onToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMode6</sender>
   <signal>toggled(bool)</signal>
   <receiver>CMainWindow</receiver>
   <slot>onToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoadPak</sender>
   <signal>triggered()</signal>
   <receiver>CMainWindow</receiver>
   <slot>onLoadPak()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMode7</sender>
   <signal>toggled(bool)</signal>
   <receiver>CMainWindow</receiver>
   <slot>onToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEnableTextures</sender>
   <signal>toggled(bool)</signal>
   <receiver>CMainWindow</receiver>
   <slot>onToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDrawBoundingBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>CMainWindow</receiver>
   <slot>onToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDrawCollision</sender>
   <signal>toggled(bool)</signal>
   <receiver>CMainWindow</receiver>
   <slot>onToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDrawPoints</sender>
   <signal>toggled(bool)</signal>
   <receiver>CMainWindow</receiver>
   <slot>onToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWireframe</sender>
   <signal>toggled(bool)</signal>
   <receiver>CMainWindow</receiver>
   <slot>onToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>materialSetComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CMainWindow</receiver>
   <slot>onMaterialSetChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>tabCloseRequested(int)</signal>
   <receiver>CMainWindow</receiver>
   <slot>onTabClosed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onExport()</slot>
  <slot>onToggled(bool)</slot>
  <slot>onLoadPak()</slot>
  <slot>onMaterialSetChanged(int)</slot>
  <slot>onTabClosed(int)</slot>
 </slots>
</ui>
